release notes/v0.38.0.md

```diff
@@ -0,0 +1,275 @@
+k6 v0.38.0 is here! 🎉
+
+## New Features!
+
+### Accessing the consolidated and derived options from the default function ([2493](https://github.com/grafana/k6/pull/2493))
+
+The `k6/execution` core module now allows accessing the consolidated and derived options k6 computed before starting the execution of the test. Consolidated options can be accessed through the `exec.test.option` property. Note that consolidated options are frozen and cannot be modified.
+
+```javascript
+import exec from 'k6/execution';
+
+export const options = {
+  vus: 10,
+  duration : '30s'
+};
+
+export default function() {
+  console.log(exec.test.options.scenarios.default.vus) // 10
+}
+```
+
+### Dumping SSL keys to an NSS formatted key log file ([2487](https://github.com/grafana/k6/pull/2487))
+
+Add the ability to dump SSL keys while making TLS connections, which then can be used to decrypt the whole traffic k6 generates. 
+
+This is done through setting the `SSLKEYLOGFILE` to some file path that and running k6. This will populate the file with the keys, and then you can use Wireshark to capture the traffic, decrypt it and use that for debugging. 
+
+You can look at how this is done with curl [here](https://daniel.haxx.se/blog/2018/01/15/inspect-curls-tls-traffic/).
+
+### Event loop testing
+
+[Exporting the event loop](https://pkg.go.dev/go.k6.io/k6/js/eventloop) added in v0.37.0, this lets extensions developers to use it to test event loop dependant APIs. Updates to [modulestest.VU](https://pkg.go.dev/go.k6.io/k6/js/modulestest#VU) to support the new API were also made.
+
+```go
```

Comment:
Maybe link to the code in the k6 repo as well? just so someone can go look at it if they want :thinking: